service-app/internal/api/handler.go

```diff
@@ -226,7 +240,21 @@ func (c *IndexController) IngestHandler(w http.ResponseWriter, r *http.Request)
 	// Send the UID response
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(http.StatusAccepted)
-	if err := json.NewEncoder(w).Encode(scannedCaseResponse); err != nil {
+
+	uid := scannedCaseResponse.UID
+	if matched, err := regexp.MatchString("^7[0-9]{3}-[0-9]{4}-[0-9]{4}$", uid); matched && err == nil {
+		uid = strings.ReplaceAll(uid, "-", "")
+	}
```

Comment:
Awkward one, but Sirius returns a UID like `7000-1234-1234` and the scanning app should return `700012341234`. It's easier to handle it here because the Sirius side is shared code with a bunch of other endpoints that need the format with dashes.

Reply:
I know its picky, we could precompiled regex for a slight performance boost where possible, I often overlook

```suggestion
	uid := scannedCaseResponse.UID
	if uidRegex.MatchString(uid) {
		uid = strings.ReplaceAll(uid, "-", "")
	}
```